go.mod

```diff
@@ -11,6 +10,7 @@ go 1.19
 require (
 	github.com/buger/jsonparser v1.1.1
 	github.com/google/uuid v1.3.0
+	github.com/layer5io/meshkit v0.6.48
 	github.com/layer5io/meshkit v0.6.50
```

Comment:
Why are we importing different versions of same package?

As the package is maintained by us only, can we not make changes to meshkit to fix this issue?

Reply:
And, why importing an older version?